### PR TITLE
feat: displayed GTFS Schedule feed if there's no feedname

### LINKED ISSUE #606: Related Schedule Feeds section: have default feed name for feeds with no dataset
### Describe the problem

Related Schedule feed section of realtime : Should have default reference like “GTFS Schedule feed” for cases where the schedule feed doesn’t have a name and there’s no dataset (e.g https://qa.mobilitydatabase.org/feeds/mdb-1792) 

### Proposed solution

Include "GTFS Schedule feed" as default feed name in cases where there are 0 datasets for the schedule feed

### Alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by qcdyx**
**Summary:**
Closes #606 
image

**Expected behavior:** 

Include "GTFS Schedule feed" as default feed name in cases where there are 0 datasets for the schedule feed

**Testing tips:**

Provide tips, procedures and sample files on how to test the feature.
Testers are invited to follow the tips AND to try anything they deem relevant outside the bounds of the testing tips. 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [ ] Add or update any needed documentation to the repo
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by emmambd** [comment]
@qcdyx It looks like now we're returning "GTFS Schedule feed" in -all- cases, not just in situations where there are 0 datasets: https://mobility-feeds-dev--pr-641-3p9az0i7.web.app/feeds/mdb-1609. 

Can this be changed?

**Comment by qcdyx** [comment]
@emmambd oops, I'm fixing it now.

**Comment by qcdyx** [comment]
mdb-1609
image
mdb-1792
image

**Comment by emmambd** [comment]
@qcdyx Perfect!

File: web-app/src/app/screens/Feed/AssociatedFeeds.tsx [hunk 1]
```diff
@@ -28,6 +28,9 @@ const renderAssociatedGTFSFeedRow = (
   }
   const hasFeedName =
     assocFeed.feed_name !== undefined && assocFeed.feed_name !== '';
+  const noFeedName =
+    assocFeed.feed_name === undefined || assocFeed.feed_name === '';
+  const noLatestDataset = assocFeed.latest_dataset === undefined;
```
Comment (davidgamez) [inline web-app/src/app/screens/Feed/AssociatedFeeds.tsx:33]: I think we can remove this variable in favor of `!hasFeedName`

**Comment by qcdyx** [comment]
!hasFeedName is my first choice, but it doesn't work for mdb-1609
image

**Comment by qcdyx** [comment]
@emmambd What's the logic for no feed name but has a latest dataset? (This logic is already implemented: If there's no feed name and no dataset, display GTFS Schedule Feed.)

**Comment by emmambd** [comment]
@qcdyx It'll just display the last updated value, which is enough, since it's implied that the realtime feed is the transit provider's

**Review by davidgamez — APPROVED** [review]
LGTM

**Pull request merged**